lang/performous.pot

```diff
@@ -778,575 +871,434 @@ msgstr ""
 msgid "has vocals"
 msgstr ""
 
-#: ../game/songs.cc:199
+#: ../game/songs.cc:396
 msgid "has duet"
 msgstr ""
 
-#: ../game/songs.cc:200
+#: ../game/songs.cc:397
 msgid "has guitar"
 msgstr ""
 
-#: ../game/songs.cc:201
+#: ../game/songs.cc:398
 msgid "drums or keytar"
 msgstr ""
 
-#: ../game/songs.cc:202
+#: ../game/songs.cc:399
 msgid "full band"
 msgstr ""
 
-#: ../game/songs.cc:230
+#: ../game/songs.cc:427
 msgid "random order"
 msgstr ""
 
-#: ../game/songs.cc:231
+#: ../game/songs.cc:428
 msgid "sorted by song"
 msgstr ""
 
-#: ../game/songs.cc:232
+#: ../game/songs.cc:429
 msgid "sorted by artist"
 msgstr ""
 
-#: ../game/songs.cc:233
+#: ../game/songs.cc:430
 msgid "sorted by edition"
 msgstr ""
 
-#: ../game/songs.cc:234
+#: ../game/songs.cc:431
 msgid "sorted by genre"
 msgstr ""
 
-#: ../game/songs.cc:235
+#: ../game/songs.cc:432
 msgid "sorted by path"
 msgstr ""
 
-#: ../game/songs.cc:236
+#: ../game/songs.cc:433
 msgid "sorted by language"
 msgstr ""
 
-#: ../game/webserver.cc:284
-msgid "Performous web frontend"
-msgstr ""
-
-#: ../game/webserver.cc:285
-msgid "View database"
-msgstr ""
-
-#: ../game/webserver.cc:287
-msgid "Search and Add"
-msgstr ""
-
-#: ../game/webserver.cc:288
-msgid "Sort by"
-msgstr ""
-
-#: ../game/webserver.cc:289
-msgid "Artist"
-msgstr ""
-
-#: ../game/webserver.cc:290
-msgid "Title"
-msgstr ""
-
-#: ../game/webserver.cc:291
-msgid "Language"
-msgstr ""
-
-#: ../game/webserver.cc:292
-msgid "Edition"
-msgstr ""
-
-#: ../game/webserver.cc:293
-msgid "Creator"
-msgstr ""
-
-#: ../game/webserver.cc:295
-msgid "Normal"
-msgstr ""
-
-#: ../game/webserver.cc:296
-msgid "Inverted"
-msgstr ""
-
-#: ../game/webserver.cc:297
-msgid "Update every 10 sec"
-msgstr ""
-
-#: ../game/webserver.cc:298
-msgid "Refresh database"
-msgstr ""
-
-#: ../game/webserver.cc:299
-msgid "Upcoming songs"
-msgstr ""
-
-#: ../game/webserver.cc:300
-msgid "Refresh playlist"
-msgstr ""
-
-#: ../game/webserver.cc:301
-msgid ""
-"Web interface by Niek Nooijens and Arjan Speiard, for full credits regarding "
-"Performous see /docs/Authors.txt"
-msgstr ""
-
-#: ../game/webserver.cc:302
-msgid "Search"
-msgstr ""
-
-#: ../game/webserver.cc:303
-msgid "Available songs"
-msgstr ""
-
-#: ../game/webserver.cc:304
-msgid "Search for songs"
-msgstr ""
-
-#: ../game/webserver.cc:305
-msgid "Yes"
-msgstr ""
-
-#: ../game/webserver.cc:306
-msgid "No"
-msgstr ""
-
-#: ../game/webserver.cc:308
-msgid "Move down"
-msgstr ""
-
-#: ../game/webserver.cc:309
-msgid "Set position"
-msgstr ""
-
-#: ../game/webserver.cc:310
-msgid "Remove song"
-msgstr ""
-
-#: ../game/webserver.cc:311
-msgid "Desired position of song"
-msgstr ""
-
-#: ../game/webserver.cc:312
-msgid "Cancel"
-msgstr ""
-
-#: ../game/webserver.cc:313
-msgid "Successfully removed song from playlist"
-msgstr ""
-
-#: ../game/webserver.cc:314
-msgid "Failed removing song from playlist"
-msgstr ""
-
-#: ../game/webserver.cc:315
-msgid "Successfully changed position of song"
```

Comment:
Still some messages which are in current master..

Reply:
Should be resolved now, thank you for your patience